.github/workflows/testing.yaml

```diff
@@ -87,7 +88,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8, 3.9, '3.10', 3.11]
+        python-version: [3.9, '3.10', 3.11]
```

Comment:
I think I'm getting a bit mixed up on which python versions apply to where. Why do we remove 3.8 from here and not `units-no-grpc` above?

Reply:
This was a mistake. At one point I was cleaning up 3.8 references in this workflow config because we'd partially removed it, but then later I figured out how to add it back and just overlooked this spot.